docs/development/pigs/pig-024.rst

```diff
@@ -0,0 +1,277 @@
+
+This PIG aims to set up the project policy about authorship for our citation scheme that is reminded here.
+
+Given the fact that Gammapy is licensed under a 3-clause BSD style license (see gammapy/LICENSE.rst), Gammapy can be used
+and even modified for a science project. For this modified version, the proposed authorship policy of this PIG is not
+applicable but the general citation scheme should be applied.
+
+This PIG is structured as follows: a reminder of our general citation scheme that was up-to-now only given orally and on
+our web pages; then, the authorship policy is given for each of the products associated with Gammapy, namely the intermediate
+releases, the Long Term Support (LTS) releases, the general Gammapy papers, and the contributions to the conference.
+
+2. Citation scheme
+==================================
+
+When Gammapy is used for any publication, contribution to conferences or software, authors should properly cite Gammapy.
+It is asked to cite the DOI of the used Gammapy version as well as the associated paper (e.g. the latest LTS
+release).
+
+.. note::
+    As the Gammapy community fully supports Open Science, we strongly encourage authors using Gammapy to follow
+    the FAIR4RS principles and to allow the reproducibility of their results. As consequence, we suggest always mentioning
+    the `Zenodo <https://zenodo.org/>`_ DOI or the `HAL <https://hal.archives-ouvertes.fr/>`_ SWHID identifier (associated with
+    the `Software Heritage <https://www.softwareheritage.org/>`_ archive) of the used release of Gammapy.
+
+This citation scheme with these **two references** will be given on our web pages.
+
+3. Authorship policy
+====================
+
+The Gammapy references contain a list of authors that requires to be updated with time according to a
+general policy. This section defines what is a `contributor` to Gammapy and the policy for each type of product.
+
+Definition of a `Contributor`
+-----------------------------
+
+Contributing to Gammapy can be made in different ways:
+
+- contributing to gammapy source code,
+- contributing to our documentation (rst pages, docstring, gammapy-webpage),
+- contributing to our DevOps environment (e.g. CI, docs and package deployment, GitHub configuration),
+- contributing to our associated projects (gammapy-benchmarks, gammapy-data, gammapy-extra),
+- contributing to the maintenance of Gammapy products on open-source repositories (conda-forge, PyPI, Zenodo, HAL, docker-hub, etc) or open hub (openhub, etc),
+- leading the precise tasks identified by the Lead Developers,
+- organizing and leading hands-on sessions or schools,
+- managing our social media accounts,
+- and coordinating the project by the Lead Developers.
+
+Any contribution without threshold on these topics renders a user as official Gammapy contributor. And the contributions
+are tracked using the GitHub contributions. However, the use of personal data is regulated in Europe and one should have the
```

Comment:
Concerning 

> And the contributions are tracked using the GitHub contributions

We list many things above that are not directly track-able using GitHub (e.g. zenodo, coordinating, website, etc...) or harder than just looking at contributions here (conda-forge, ...),  so I'd leave this out.

Reply:
Indeed, the sentence is too strict. The -> Many